src/workloads/query/multiplanner/NoGoodPlan.yml

```diff
@@ -1,15 +1,16 @@
 SchemaVersion: 2018-07-01
 Owner: "@mongodb/query"
 Description: |
-  The goal of this test is to exercise multiplanning. We create 64 indexes and run a query that
-  makes all of them eligible, so we get as many competing plans as possible. The only selective
-  field is unindexed, however, meaning no index will be effective in planning. By ensuring all plans
-  are relatively equally bad, we are likely to hit the works limit sooner than the 101 results
-  limit.
+  The goal of this test is to exercise the case in multiplanning where all competing plans are bad.
+  
+  As in 'Simple.yml' we create 64 indexes and run a query that makes all of them eligible, so we
```

Comment:
nit: isn't it 63 indexes ? It doesn't look like you're actually using the **_id** index (and it's created automatically).

Maybe reuse the later phrasing:

```
We create as many indexes as possible, and run a query that makes all of them eligible
```

Reply:
We ended up deleting this workload, in another PR: https://github.com/mongodb/genny/pull/1226a